app/controllers/hyrax/pages_controller.rb

```diff
@@ -10,7 +10,7 @@ module Hyrax
   # Shows the about and help page
   class PagesController < ApplicationController
     load_and_authorize_resource class: ContentBlock, except: :show
-    layout :pages_layout
+    layout 'dc_repository_about'
```

Comment:
Reading Notes: In the previous implementation, we were using the `pages_layout` method to switch the layout based on the action name.  In this implementation, we're saying "always use the dc_repository_about" layout regardless of the action.

```ruby
      def pages_layout
        action_name == 'show' ? 'homepage' : 'hyrax/dashboard'
      end
```

My suspicion is that for the `show` we may still want to use the `homepage` layout.

Reply:
This has been updated to use the symbol for pages_layout.